diff --git a/src/Entity/MiamiHighlight.php b/src/Entity/MiamiHighlight.php
index 61656e3..c0e092e 100644
--- a/src/Entity/MiamiHighlight.php
+++ b/src/Entity/MiamiHighlight.php
@@ -2,10 +2,12 @@
 
 namespace App\Entity;
 
+use ApiPlatform\Core\Annotation\ApiResource;
 use Doctrine\ORM\Mapping as ORM;
 
 /**
  * @ORM\Entity(repositoryClass="App\Repository\MiamiHighlightRepository")
+ * @ApiResource()
  */
 class MiamiHighlight
 {
