diff --git a/src/Entity/MiamiHighlight.php b/src/Entity/MiamiHighlight.php
new file mode 100644
index 0000000..1c984f3
--- /dev/null
+++ b/src/Entity/MiamiHighlight.php
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity(repositoryClass="App\Repository\MiamiHighlightRepository")
+ */
+class MiamiHighlight
+{
+    /**
+     * @ORM\Id
+     * @ORM\GeneratedValue
+     * @ORM\Column(type="integer")
+     */
+    private $id;
+
+    // add your own fields
+}
diff --git a/src/Repository/MiamiHighlightRepository.php b/src/Repository/MiamiHighlightRepository.php
new file mode 100644
index 0000000..c2780ce
--- /dev/null
+++ b/src/Repository/MiamiHighlightRepository.php
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Repository;
+
+use App\Entity\MiamiHighlight;
+use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
+use Symfony\Bridge\Doctrine\RegistryInterface;
+
+class MiamiHighlightRepository extends ServiceEntityRepository
+{
+    public function __construct(RegistryInterface $registry)
+    {
+        parent::__construct($registry, MiamiHighlight::class);
+    }
+
+    /*
+    public function findBySomething($value)
+    {
+        return $this->createQueryBuilder('m')
+            ->where('m.something = :value')->setParameter('value', $value)
+            ->orderBy('m.id', 'ASC')
+            ->setMaxResults(10)
+            ->getQuery()
+            ->getResult()
+        ;
+    }
+    */
+}
