diff --git a/src/Controller/SunshineController.php b/src/Controller/SunshineController.php
index 5dfa7da..9765c56 100644
--- a/src/Controller/SunshineController.php
+++ b/src/Controller/SunshineController.php
@@ -3,6 +3,7 @@
 namespace App\Controller;
 
 use App\Entity\MiamiHighlight;
+use App\Repository\MiamiHighlightRepository;
 use Doctrine\ORM\EntityManagerInterface;
 use Psr\Log\LoggerInterface;
 use Symfony\Bundle\FrameworkBundle\Controller\Controller;
@@ -48,7 +49,7 @@ class SunshineController extends Controller
     /**
      * @Route("/miami/{neighborhood}")
      */
-    public function miami($neighborhood, LoggerInterface $logger)
+    public function miami($neighborhood, LoggerInterface $logger, MiamiHighlightRepository $repo)
     {
         $neighborhood = ucwords(str_replace('-', ' ', $neighborhood));
 
@@ -58,9 +59,11 @@ class SunshineController extends Controller
 
         $logger->info('Talking about the neighborhood: '.$neighborhood);
 
+        $highlights = $repo->findAll();
+
         return $this->render('sunshine/miami.html.twig', [
             'neighborhood' => $neighborhood,
-            'highlights' => ['beaches', 'Cuban food', 'no snow', 'Adam Culp']
+            'highlights' => $highlights
         ]);
     }
 }
diff --git a/templates/sunshine/miami.html.twig b/templates/sunshine/miami.html.twig
index f96b219..9171879 100644
--- a/templates/sunshine/miami.html.twig
+++ b/templates/sunshine/miami.html.twig
@@ -5,7 +5,7 @@
 
     <ul>
         {% for highlight in highlights %}
-            <li>{{ highlight }}</li>
+            <li>{{ highlight.name }} ({{ highlight.description }})</li>
         {% endfor %}
     </ul>
 {% endblock %}
