diff --git a/src/EventSubscriber/UserAgentLoggerSubscriber.php b/src/EventSubscriber/UserAgentLoggerSubscriber.php
index 0cd604b..945982c 100644
--- a/src/EventSubscriber/UserAgentLoggerSubscriber.php
+++ b/src/EventSubscriber/UserAgentLoggerSubscriber.php
@@ -9,7 +9,7 @@ class UserAgentLoggerSubscriber implements EventSubscriberInterface
 {
     public function onKernelRequest(GetResponseEvent $event)
     {
-        // ...
+        dump($event);die;
     }
 
     public static function getSubscribedEvents()
