diff --git a/src/EventSubscriber/UserAgentLoggerSubscriber.php b/src/EventSubscriber/UserAgentLoggerSubscriber.php
index 5a59122..4f7d160 100644
--- a/src/EventSubscriber/UserAgentLoggerSubscriber.php
+++ b/src/EventSubscriber/UserAgentLoggerSubscriber.php
@@ -2,15 +2,23 @@
 
 namespace App\EventSubscriber;
 
+use Psr\Log\LoggerInterface;
 use Symfony\Component\EventDispatcher\EventSubscriberInterface;
 use Symfony\Component\HttpKernel\Event\GetResponseEvent;
 
 class UserAgentLoggerSubscriber implements EventSubscriberInterface
 {
+    private $logger;
+
+    public function __construct(LoggerInterface $logger)
+    {
+        $this->logger = $logger;
+    }
+
     public function onKernelRequest(GetResponseEvent $event)
     {
         $userAgent = $event->getRequest()->headers->get('User-Agent');
-        dump($userAgent);die;
+        $this->logger->info('The user agent is: '.$userAgent);
     }
 
     public static function getSubscribedEvents()
