diff --git a/src/Controller/SunshineController.php b/src/Controller/SunshineController.php
index 942720b..c921166 100644
--- a/src/Controller/SunshineController.php
+++ b/src/Controller/SunshineController.php
@@ -14,7 +14,11 @@ class SunshineController extends Controller
     {
         $neighborhood = ucwords(str_replace('-', ' ', $neighborhood));
 
-        return $this->json([
+//        return $this->json([
+//            'highlights' => ['beaches', 'Cuban food', 'no snow', 'Adam Culp']
+//        ]);
+
+        return $this->render('sunshine/miami.html.twig', [
             'neighborhood' => $neighborhood,
             'highlights' => ['beaches', 'Cuban food', 'no snow', 'Adam Culp']
         ]);
diff --git a/templates/sunshine/miami.html.twig b/templates/sunshine/miami.html.twig
new file mode 100644
index 0000000..f96b219
--- /dev/null
+++ b/templates/sunshine/miami.html.twig
@@ -0,0 +1,11 @@
+{% extends 'base.html.twig' %}
+
+{% block body %}
+    <h1>Hiya {{ neighborhood }}, Miami!</h1>
+
+    <ul>
+        {% for highlight in highlights %}
+            <li>{{ highlight }}</li>
+        {% endfor %}
+    </ul>
+{% endblock %}
